ccd-definition/CaseField.json

```diff
@@ -85,7 +85,7 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "Shared_Storage_DRAFTType",
     "ID": "documents_socialWorkChronology_document",
-    "Label": "Social work chronology",
+    "Label": " ",
```

Comment:
how do blank labels appear in the ui?

Reply:
I talked this over with james if you dont have a blank it does not like it so adding the blank space just allows it to render properly I will add a screenshot below 
<img width="701" alt="screenshot 2018-12-07 at 14 16 33" src="https://user-images.githubusercontent.com/42037338/49652705-be1de200-fa2a-11e8-9309-44d351e7fc48.png">

Reply:
Also @kurt-uk here are some screenshots
<img width="1423" alt="screenshot 2018-12-07 at 14 14 48" src="https://user-images.githubusercontent.com/42037338/49652748-d7269300-fa2a-11e8-9850-e0c868a2be97.png">
![Uploading Screenshot 2018-12-07 at 14.14.59.png…]()
![Uploading Screenshot 2018-12-07 at 14.15.11.png…]()

Reply:
you need to wait until they all upload 😉

Reply:
> I talked this over with james if you dont have a blank it does not like it so adding the blank space just allows it to render properly I will add a screenshot below
> <img alt="screenshot 2018-12-07 at 14 16 33" width="701" src="https://user-images.githubusercontent.com/42037338/49652705-be1de200-fa2a-11e8-9309-44d351e7fc48.png">

does the label in caseField accept markdown? We could remove the larger labels and instead put them instead of " ". Less lines and then no blanks. Thoughts?

Reply:
they should do

Reply:
<img width="1122" alt="screenshot 2018-12-07 at 13 52 53" src="https://user-images.githubusercontent.com/42037338/49653488-02aa7d00-fa2d-11e8-8af2-a3099213ed6c.png">
<img width="1423" alt="screenshot 2018-12-07 at 14 14 48" src="https://user-images.githubusercontent.com/42037338/49653489-02aa7d00-fa2d-11e8-957e-cd37ae2d67de.png">
<img width="1431" alt="screenshot 2018-12-07 at 14 14 59" src="https://user-images.githubusercontent.com/42037338/49653490-02aa7d00-fa2d-11e8-8d93-9073aa2d60a8.png">

Reply:
Sam, is it maybe better to do this as tech debt? There's a few instances of empty labels in caseField.json.

Reply:
Changes made and pushed @swalker125 do you want to review

Reply:
<img width="1423" alt="screenshot 2018-12-07 at 14 59 08" src="https://user-images.githubusercontent.com/42037338/49654801-bcefb380-fa30-11e8-9cf2-e8105fc99f93.png">
<img width="630" alt="screenshot 2018-12-07 at 14 59 22" src="https://user-images.githubusercontent.com/42037338/49654803-bcefb380-fa30-11e8-830e-48d4b8388f66.png">
<img width="609" alt="screenshot 2018-12-07 at 14 59 28" src="https://user-images.githubusercontent.com/42037338/49654804-bcefb380-fa30-11e8-9117-470dd9e86e26.png">
<img width="652" alt="screenshot 2018-12-07 at 14 59 35" src="https://user-images.githubusercontent.com/42037338/49654805-bd884a00-fa30-11e8-9c40-4fbba5ee3115.png">

Reply:
Hashtags removed from documents screen 
<img width="948" alt="screenshot 2018-12-07 at 16 15 13" src="https://user-images.githubusercontent.com/42037338/49659075-515f1380-fa3b-11e8-919d-dff02cff03cf.png">